src/components/TypeLayout/_mixins.scss

```diff
@@ -62,11 +62,13 @@
     }
 
     &--sm {
+      @include carbon--type-style($name: caption-01);
```

Comment:
It looks like there's some duplication here - We can reduce that by grouping the selectors

Reply:
👍 Done